### PR TITLE
Fixing errors with mongoDB and python's version in manual scoring, and a error with rendering notebooks

### LINKED ISSUE #278: Students see manual grading feedback
**Is your feature request related to a problem? Please describe.**
When manual scoring plugin is set, the students cannot see the manual scoring feedback

**Describe the solution you'd like**
When a student enter to a course, right below the `my statistics` button, we can add the access to the page where the students see their feedback. Another option might be when the students open a task that is closed, they may see their feedback.

**Additional context**
We have to wait until the PR related to the manual scoring new feature is merged.

### PR DESCRIPTION
**Pull request by Heldeg**
# Description

This PR fixs the problem with a unsupported command for our mongoDB's version, `$mergeObjects`, a problem with string format in python due to python version and a problem when a notebook task was displayed.

Fixes #278 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

- [ ] The DB request return the necessary information
- [ ] The title `Manual Scoring` looks good in the menu in English and Spanish
- [ ] Feedback and Manual Scoring pages can show the code of a notebook

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings

### CODE REVIEW

**Comment by crdgonzalezca** [comment]
Just a question, why 100 commits? probably this is an outdated branch. Try to squash them https://www.internalpointers.com/post/squash-commits-into-one-git

**Review by crdgonzalezca — APPROVED** [review]
Please squash the commits as mentioned in the other comment.

File: inginious/frontend/plugins/manual_scoring/static/js/code_field.js [hunk 1]
```diff
@@ -1,52 +0,0 @@
-const LANGUAGES = {
```
Comment (crdgonzalezca) [inline inginious/frontend/plugins/manual_scoring/static/js/code_field.js:1]: Why did you remove this? Check again after these last changes that everything is working ok.

Comment (Heldeg) [reply inginious/frontend/plugins/manual_scoring/static/js/code_field.js:1]: A few weeks ago I had problem minifying files and the problem was related to the content of that file. The file had worked normally (without min files) and I hadn't modified a lot, therefore I deleted, modify the file, the name, etc. To know what was the matter. The problem was how I declared a variable, but with the updates with master, two files stayed. The content is the same, just changed the name of the class.

Comment (crdgonzalezca) [reply inginious/frontend/plugins/manual_scoring/static/js/code_field.js:1]: So there is no problem right? can I merge this without introducing a bug?

Comment (Heldeg) [reply inginious/frontend/plugins/manual_scoring/static/js/code_field.js:1]: Yes
